nornir/core/inventory.py

```diff
@@ -150,6 +162,12 @@ def dict(self) -> Dict[str, Any]:
             **super().dict(),
         }
 
+    def add_to_group(self, group: "Group") -> None:
```

Comment:
I think this method is a bit unnecessary. In the original issue it was more interesting because that required also adding a reference but now that’s not needed so:

```
host.groups.add(group)
```

or

```
host.add_to_group(group)
```

do pretty much the same and the first is actually shorter to type :P

Reply:
Makes sense.  it gets a lot easier without the ref.